### PR TITLE
[MTL+] add av1-8bit non low power mode support

### PR DESCRIPTION
**Pull request by FocusLuo**
add av1-8bit non low power mode support for MTL+ platforms

### CODE REVIEW

**Review by uartie — CHANGES_REQUESTED** [review]
I have same concerns as https://github.com/intel/vaapi-fits/pull/557

File: test/ffmpeg-qsv/encode/av1.py [hunk 1]
```diff
@@ -6,11 +6,97 @@
...
+      bframes   = bframes,
+      qp        = qp,
+      rcmode    = "cqp",
+      quality   = quality,
+      profile   = profile,
+      tilerows  = tilerows,
+      tilecols  = tilecols,
+    )
+
+  @slash.parametrize(*gen_av1_cqp_lp_parameters(spec))
```
Comment (uartie) [inline test/ffmpeg-qsv/encode/av1.py:29]: This is the wrong parameter generator... it is getting config from the cqp_lp variant.  Please add a new generator that generates cases from the "cqp" variants.

Also, please do the same for the others (cbr, vbr).

**Review by uartie — CHANGES_REQUESTED** [review]
wrong case variant generators.

File: test/ffmpeg-qsv/encode/av1.py (hunk 1 above)
Comment (FocusLuo) [reply test/ffmpeg-qsv/encode/av1.py]: updated

**Review by uartie — APPROVED** [review]
LGTM

**Pull request merged**